docs/wallet_API.md

```diff
@@ -6,27 +6,27 @@ author: Claude Barde
 
 ## What is the Wallet API?
 
-The Tezos blockchain is a fantastic tool, but it is self-contained. Except for the transactions you send to it, it has no interaction with the outside world. However, it would be amazing to interact with it, such as making payments, reading the balance of an account, or recording data in a smart contract. This interaction is what the Wallet API (and Taquito in general) achieves. The Wallet API provides a new yet familiar way to interact with the blockchain and smart contracts by delegating several actions that Taquito previously handled to the wallets. This delegation offers more flexibility for both developers and users and gives the ecosystem more space to evolve.
+You have learned how to use Taquito to interact with the Tezos blockchain. Up to this document, you have been using a signer to sign operations. For interactive dApps, it is common to use a wallet to sign operations. From a user's perspective, the workflow is as follows:
```

Comment:
This seems an assumptions, fe dapp dev can skip entire contractAPI and directly pick up walletAPI
> You have learned how to use Taquito to interact with the Tezos blockchain. Up to this document, you have been using a signer to sign 

operations.

👏 Love the addition of workflow explanation!

Reply:
I kind-of had the assumption that anyone who reads this code, has at least read the quickstart (or knows most of it).